### PR TITLE
Fix issue with ac_present not being updated on AC Power Lost

### PR DESCRIPTION
**Pull request by tomachristian**
This is a bug that I noticed in Home Assistant and then found in code. I am not familiar with Python or the code base, so I don't know which parts need an update now or how to add a (proper) test for this.

### CODE REVIEW

File: pyenvisalink/dsc_client.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def handle_keypad_update(self, code, data):
             for i in range(0, 7):
                 if bits[15-i] == '1':
                     trouble_description += evl_verboseTrouble[i] + ', '
-                    if i == 2:
+                    if i == 1:
```
Comment (tomachristian) [inline pyenvisalink/dsc_client.py:201]: `ac_present` was not being updated to `False` when `bit 1 = AC Power Lost` as per [TPI docs](http://forum.eyez-on.com/FORUM/download/file.php?id=223&sid=9e527acd838441b919657c5d636fd852), but on `bit 2 = Telephone Line Fault`. That is why many people had/have problems with `ac_present` not being set to `False` in Home Assistant when AC power is lost.

cc @Cinntax

Comment (Cinntax) [reply pyenvisalink/dsc_client.py:201]: Hi Tom- apologies for the delay here.  Looks good to me- especially if you've had a chance to test in your environment.
That being said- in order for this change to work with HA, I'll need to make a new release of this library, and then a pull request on Home-assistant will need to be made to update the dependencies so it'll pull the updated version of this library.

II'm in the process of moving, and i'm temporarily not running home-assistant - so i'm not really able to verify anything on that front- so if you could do the PR over there that'd be great.

Comment (tomachristian) [reply pyenvisalink/dsc_client.py:201]: Yo @Cinntax, if you can do the release then I'll create the update PR on HA side. Thank you! Good luck with moving!

Comment (dancinnamon-okta) [reply pyenvisalink/dsc_client.py:201]: Thanks @tomachristian -
Sorry for the delay- version 4.1 has been deployed to pypi.

**Pull request merged**